### PR TITLE
Boardgame list

### PR DESCRIPTION
**Pull request by peoplenarthax**
boardgame-list

Added boardgame list element as in figma design https://www.figma.com/file/l1gs29hkHyFcLvqVeEEnuV/Create-match?node-id=1%3A2

Decided to go for styled-icons as tree shakable, compatible with styled components and avoiding adding core element and different styling system from material-ui

For dates date-fns is tree shakable and as long as we do not add locales it feels pretty small.

Next step:
 Embed this with search bar in the select boardgame view. 
 Server to send the last date you played a boardgame.
 Add click stylings

### CODE REVIEW

**Comment by beeequeue** [comment]
The icons seem to have some padding on the left

File: packages/ui/src/components/animated-list/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export const AnimatedList : React.FC = ({children} ) => {
+  const elements = Array.isArray(children) ? children : [children]
+  const trail = useTrail(elements.length, trailConfig)
+  return (
+    <ListComponent>
+      {trail.map(({ x, ...rest }, index) => (
+        <animated.div
+          key={index}
+          style={{
+            ...rest, transform: (x as OpaqueInterpolation<number>).interpolate(x => `translate3d(0,${x}px,0)`)
```
Comment (beeequeue) [inline packages/ui/src/components/animated-list/index.tsx]: Can use `translateY` instead here

File: yarn.lock [hunk 2]
```diff
@@ -2884,6 +2903,134 @@
...
+
+"@styled-icons/icomoon@^9.0.1":
+  version "9.0.1"
+  resolved "https://registry.yarnpkg.com/@styled-icons/icomoon/-/icomoon-9.0.1.tgz#43b97acf25f89aaf78918b968142d867bf2efaba"
+  integrity sha512-PHXCeq95ODVEz4TpEZKlKwOcWKRRqnFmqi6iqcD++EMaNfUQvWSvs0aP+724VqfkPrNmyTFvjn16toZlTAHrsg==
+  dependencies:
+    "@styled-icons/styled-icon" "^9.0.1"
+    tslib "^1.9.3"
+
+"@styled-icons/material@^9.0.1":
```
Comment (beeequeue) [inline yarn.lock]: should we only install the ones we need? i.e. only `@styled-icons/material` so far?
probably isnt really necessary

**Comment by beeequeue** [comment]
looks very pretty otherwise :)

**Comment by beeequeue** [comment]
no border radius on the images

**Comment by peoplenarthax** [comment]
> The icons seem to have some padding on the left

They had a small padding that in a small phone is not visible. I added a bit of margin to the titles as i am not able to modify the icons

File: yarn.lock (hunk 2 above)
Comment (peoplenarthax) [reply yarn.lock]: Yes, forgot to remove the rest of the library as i was experimenting with other icons

File: packages/ui/src/components/animated-list/index.tsx (hunk 1 above)
Comment (peoplenarthax) [reply packages/ui/src/components/animated-list/index.tsx]: better!

**Comment by beeequeue** [comment]
better to leave the padding on the icons than add margin to the title IMO

**Comment by peoplenarthax** [comment]
Agreed, I will remove that now and move. If that becomes to annoying we will fix it

**Review by beeequeue — APPROVED** [review]

**Pull request merged**